src/components/MDXComponents/__tests__/MDXLink.test.tsx

```diff
@@ -0,0 +1,153 @@
+import { render, screen } from '@testing-library/react';
+import { MDXLink } from '../MDXLink';
+
+const routerMock = {
+  __esModule: true,
+  useRouter: () => {
+    return {
+      query: {
+        platform: ''
+      },
+      asPath: ''
+    };
+  }
+};
+
+jest.mock('next/router', () => routerMock);
+//   const MockedNextLink = ({ children, href }) => {
```

Comment:
Just a nit to delete this commented out stuff

Reply:
Thank you for catching this! Sorry about that, for some reason in my vscode, the line above it was collapsed so I thought I deleted it already (it was hiding all these lines instead).